### DISCUSSION TITLE
Azure DevOps Pipeline PowerPages Fails

### DISCUSSION THREAD

**Discussion by mikevalentinecanteen** [microsoft/powerplatform-build-tools#897] [Q&A]
Within the last week or two, our PowerPages pipeline has started failing.  It is giving the following error:

Error: Power Pages entities for standard data model (v1) aren't available on this org, please install required packages on this org. If you are trying to upload to enhanced data model (v2), pass model version as "2". For more information see https://learn.microsoft.com/en-us/power-pages/admin/enhanced-data-model#data-model-power-platform-cli-parameters .

It's a V1 portal.  This has worked for ~2 years and of course the Portal solutions are installed ;).  Full error log attached.

[PortalFailure.txt](https://github.com/microsoft/powerplatform-build-tools/files/15438831/PortalFailure.txt)

**Comment by tehcrashxor** [comment]
The PowerPages team has provided a fix in the just-released PAC CLI v1.32.8.
Power Platform Build Tools [v2.0.71](https://github.com/microsoft/powerplatform-build-tools/releases/tag/v2.0.71) is deploying right now with that update.

**Reply by mikevalentinecanteen** [reply]
This has started happening again today.  I think it was probably something in the bump PAC to 1.33.4 commit that was done yesterday.  I had DevOps pipelines run successfully yesterday morning and now they are failing with this same error again.

**Reply by vikaskumarsingh123** [reply]
Hey @mikevalentinecanteen Did you end up finding a solution to this? This has recurred in the latest Build Tools.

Last time the Pipelines ran successfullly was with PAC < 1.33.4

I am currently on Power Platform Build Tools v2.0.77 which has PAC CLI v1.34.4
